### DISCUSSION TITLE
Ability to resize split views just by dragging.

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#768] [Ideas]
### What feature would you like to see?

I generally split view 2 tabs and I would like to resize them just by dragging. 

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/679_

**Comment by MikolajQ** [comment]
Yes, much desired feature!!!

**Comment by Tkdefender88** [comment]
Is it possible to resize the splits at all? They seem completely fixed. They don't seem to split evenly either

**Comment by kittybwained** [comment]
This would be incredibly useful, please add this

**Comment by universish** [comment]
This is not a feature of zen-browser. At least that's how I know it, it's a feature that comes with the installation of a mod (I don't remember which mod, if you do, please let me know)
Please move this topic to the github page of the mod you installed.

**Comment by XanderLeaDaren** [comment]
This feature is currently provided (version 1.10.1b), discussion may be closed for better search results visibility. 😊
